### PR TITLE
README.md: update

### PR DESCRIPTION
**Pull request by toobuntu**
Thank you for setting up this tap! I see there are some default files in place.

This PR aims to spruce up the readme.

Query whether it makes sense to include an Intel installation in this tap. Users are probably better off just using the official Homebrew cask for that. If the purpose of this tap is to support Apple Silicon, then it could be fine if the cask here supports only arm64 (in which case the footnote in the readme should be deleted).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,65 @@
...
+    ```
+
+    Or, if you prefer step-by-step:
+
+    ```bash
+    # Add the FreeTube tap (required only once).
+    brew tap pikachuexe/freetube
+
+    # Install FreeTube from the tap.
+    brew install --no-quarantine pikachuexe-freetube
```
Comment (PikachuEXE) [inline README.md:31]:
```suggestion
    brew install --no-quarantine freetube
```

File: README.md [hunk 2]
```diff
@@ -1,11 +1,65 @@
...
+
+
+#### Quick Installation
+
+1. Ensure [Homebrew](https://brew.sh) is installed.
+
+1. Install FreeTube with a single command:
+
+    ```bash
+    brew install --no-quarantine pikachuexe/freetube/pikachuexe-freetube
```
Comment (PikachuEXE) [inline README.md:21]:
```suggestion
    brew install --no-quarantine pikachuexe/freetube/freetube
```

**Review by PikachuEXE — CHANGES_REQUESTED** [review]
The formula is still `freetube` (I tested it yesterday and just now)

**Comment by toobuntu** [comment]
See #2. There is a name collision between this tap and homebrew-cask. This is disallowed.

> If a token conflicts with an already-existing cask, authors should manually make the new token unique by prepending the vendor name.[^collision]

[^collision]: https://github.com/Homebrew/brew/blob/master/docs/Adding-Software-to-Homebrew.md#cask-token-details

**Pull request merged**